### ISSUE TITLE
Feature Request - Add ability in Workflows to select a "Send From" email address

### ISSUE THREAD

**Issue by cjmny** [salesagility/SuiteCRM#1784]
#### Issue

We use Workflows for generating emails in many situations in our CRM.  For example, if we are performing an evaluation of a business, we send an email to the owner of the business, a group employees here, and the evaluator.  Another example is in our lead generation.  When a lead is entered, we use the Workflow to send an email to the lead immediately.  In these two situations, we would want the emails coming from different email addresses (a current customer doesn't need to see the "sales" email, and we don't necessarily want leads sending emails in to non sales personnel).  
#### Expected Behavior

It would be great if the workflow could allow the user to set the "from" address and name in some way.  This would allow us to simplify our email notification process.
#### Actual Behavior

All emails get sent from one address.
#### Steps to Reproduce

N/A
#### Your Environment

Suite 7.6.5

**Comment by vogelap** [comment]
+1

-drew 
www.drewvogel.com

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Feature Request - Add ability in Workflows to select a "Send From" email address](https://trello.com/c/ORrrn5RM/83-feature-request-add-ability-in-workflows-to-select-a-send-from-email-address)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
